public/server/db/lmao.json

```
@@ -0,0 +1,3 @@
+{
+  "definition": "Laughing My Ass Out"
```

Is `Laughing my ass off` not more common?
```suggestion
  "definition": "Laughing My Ass Off"
```

---

You're right. "off" is more commonly used. I should add "out" as an alternative.